packages/stencil/src/executors/stencil-runtime/stencil-config.ts

```diff
@@ -44,7 +44,10 @@ export async function initializeStencilConfig<
     logger.enableColors(false);
   }
 
-  const projectDir = context.workspace.projects[context.projectName].root;
+  const projects = context.workspace
+    ? context.workspace.projects
+    : context.projectsConfigurations.projects;
```

Comment:
What about only using `context.projectsConfigurations.projects` as we don't want to support older Nx versions?

Reply:
Sure, done